README.md

```diff
@@ -94,25 +91,6 @@ the internet. If redeploying the infrastructure to another region, **make sure t
 There can only be one deployment bound to the domain at any given time. Destroying the old deployment before creating
 the new one will result in downtime.
 
-### TLS certificates
```

Comment:
Until 23rd of April, we had these certs config files. We stopped using them at around the 10th of April when we moved from nginx to aws load balancer controller.